data/35e/wizards_of_the_coast/core/dungeon_masters_guide/dmg_equip.lst

```diff
@@ -596,7 +596,7 @@ Ring of Force Shield															OUTPUTNAME:[NAME]								TYPE:Magic.Ring	COST
 Ring of Freedom of Movement														OUTPUTNAME:[NAME]								TYPE:Magic.Ring	COST:40000	WT:0								SOURCEPAGE:p.232												SPELLS:Magic Item|TIMES=ATWILL|CASTERLEVEL=7|Freedom of Movement																														SPROP:This gold ring allows the wearer to act as if continually under the effect of a freedom of movement spell. Moderate abjuration; CL 7th
 Ring of Friend Shield															OUTPUTNAME:[NAME]								TYPE:Magic.Ring	COST:50000	WT:0								SOURCEPAGE:p.232																																																				SPROP:These curious rings always come in pairs. A friend shield ring without its mate is useless. Either wearer of one of a pair of the rings can, at any time, command his or her ring to cast a shield other spell with the wearer of the mated ring as the recipient. This effect has no range limitation. Moderate abjuration; CL 10th
 Ring of Friend Shield (1 of 2)													OUTPUTNAME:[NAME]								TYPE:Magic.Ring	COST:25000	WT:0								SOURCEPAGE:p.232																																																				SPROP:These curious rings always come in pairs. A friend shield ring without its mate is useless. Either wearer of one of a pair of the rings can, at any time, command his or her ring to cast a shield other spell with the wearer of the mated ring as the recipient. This effect has no range limitation. Moderate abjuration; CL 10th
-Ring of Invisibility															OUTPUTNAME:[NAME]								TYPE:Magic.Ring	COST:20000	WT:0								SOURCEPAGE:p.232												SPELLS:Magic Item|TIMES=ATWILL|CASTERLEVEL=3|Invisibility																															SPROP:By activating this simple silver ring, the wearer can benefit from invisibility, as the spell (PH 245). Faint illusion; CL 3rd
+Ring of Invisibility															OUTPUTNAME:[NAME]								TYPE:Magic.Ring	COST:20000	WT:0								SOURCEPAGE:p.232												SPELLS:Magic Item|TIMES=ATWILL|CASTERLEVEL=3|Invisibility																															SPROP:By activating this simple silver ring, the wearer can benefit from invisibility, as the spell (PH 245)
```

Comment:
Why is the faint illusion; CL 3rd being removed?

Reply:
that was in error, corrected.